### PR TITLE
updates for new data fields

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/785)

### CODE REVIEW

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx [hunk 1]
```diff
@@ -135,11 +135,11 @@ const EmployeeAgendaSearchCard = ({ isCDO, result, showCreate, viewType }) => {
             (showAgendaItemMaintenance && agendaIDExist) ?
               <dd>
                 <Link to={`/profile/${userRole}/createagendaitem/${perdet}/${agendaID}`} className="agenda-edit-button">
-                  {agendaStatus}
+                  {agendaStatus || <FA name="sticky-note-o" />}
                 </Link>
               </dd>
               :
-              <dd>{agendaStatus}</dd>
+              <dd>{agendaStatus || FALLBACK}</dd>
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:142]: do we want to put the FALLBACK in the declaration above for consistency?

File: src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx [hunk 2]
```diff
@@ -129,11 +121,11 @@ const EmployeeAgendaSearchRow = ({ isCDO, result, showCreate, viewType }) => {
               (showAgendaItemMaintenance && agendaIDExist) ?
                 <dd>
                   <Link to={`/profile/${userRole}/createagendaitem/${perdet}/${agendaID}`} className="agenda-edit-button">
-                    {agendaStatus}
+                    {agendaStatus || <FA name="sticky-note" />}
                   </Link>
                 </dd>
                 :
-                <dd>{agendaStatus}</dd>
+                <dd>{agendaStatus || FALLBACK}</dd>
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx:128]: same question as [here](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2520/files#r1149709908)

File: src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx [hunk 3]
```diff
@@ -100,14 +100,6 @@ const EmployeeAgendaSearchRow = ({ isCDO, result, showCreate, viewType }) => {
             <dt>CDO:</dt>
             <dd>{cdo}</dd>
           </div>
-          {/*
```
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx:103]: Author was a great show, sorry to see it go😢

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx [hunk 4]
```diff
@@ -135,11 +135,11 @@ const EmployeeAgendaSearchCard = ({ isCDO, result, showCreate, viewType }) => {
             (showAgendaItemMaintenance && agendaIDExist) ?
               <dd>
                 <Link to={`/profile/${userRole}/createagendaitem/${perdet}/${agendaID}`} className="agenda-edit-button">
-                  {agendaStatus}
+                  {agendaStatus || <FA name="sticky-note-o" />}
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:138]: which note should i choose? diff for row and card to show

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:142]: if we do it won't work for L138

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx (hunk 4 above)
Comment (SidelineCory24) [inline src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:138]: 🥳

File: src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx (hunk 3 above)
Comment (elizabeth-jimenez) [reply src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx:103]: Did u make that rhyme on purpose? It seems like, on the surface.

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx (hunk 1 above)
Comment (SidelineCory24) [reply src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:142]: yep, just saw

File: src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx (hunk 2 above)
Comment (SidelineCory24) [reply src/Components/Agenda/EmployeeAgendaSearchRow/EmployeeAgendaSearchRow.jsx:128]: question was answered above

**Comment by SidelineCory24** [comment]
Looks good, will circle back once mock pr is up

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx [hunk 5]
```diff
@@ -135,11 +135,11 @@ const EmployeeAgendaSearchCard = ({ isCDO, result, showCreate, viewType }) => {
             (showAgendaItemMaintenance && agendaIDExist) ?
               <dd>
                 <Link to={`/profile/${userRole}/createagendaitem/${perdet}/${agendaID}`} className="agenda-edit-button">
-                  {agendaStatus}
+                  {agendaStatus || "No Status" />}
```
Comment (scott062) [inline src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:138]: 👀

**Review by scott062 — APPROVED** [review]
🚀  Very nice />

**Comment by elizabeth-jimenez** [comment]
> > 🚀 Very nice />
> 
> are you on the right branch?

How about now?

**Comment by elizabeth-jimenez** [comment]
> > > 🚀 Very nice />
> > 
> > 
> > are you on the right branch?
> 
> How about now?

Check once more, please

**Pull request merged**